lib/bin/command-proxy-client.js

```diff
@@ -10,44 +10,43 @@
 // ----- OUTPUT -----
```

Comment:
Just be aware, babelizing this one might create some weird issues, since it is called from within Instruments, and we were only using core node depencies up to now.